bothub_nlp_api/tests/test_api_v2/test_debug_parse_api.py

```diff
@@ -0,0 +1,50 @@
+import unittest
+from starlette.testclient import TestClient
+from bothub_nlp_api import app as api
+from unittest.mock import patch
+
+
+class TestDebugParseRoute(unittest.TestCase):
+    def setUp(self):
+        self.app = TestClient(api.app)
+        self.header = {
+            'Authorization': 'Bearer da39f8a1-532a-459e-85c0-bfd8f96db828'
```

Comment:
Esse token é um token válido ? se for, o ideal seria você fazer mock das requests

Reply:
Acredito que não, mas vou alterar por via das duvidas